src/pages/manager/OpenAccountPage.js

```diff
@@ -1,11 +1,52 @@
 const { expect } = require('@playwright/test');
 
-export class AddCustomerPage {
+export class OpenAccountPage {
   constructor(page) {
-    this.page = page; 
+    this.page = page;
+    this.customerNameDropdown = page.getByTestId('userSelect'); 
+    this.currencyDropdown = page.getByTestId('currency');     
+    this.processButton = page.getByRole('button', { name: 'Process' }); 
   }
 
   async open() {
     await this.page.goto('/angularJs-protractor/BankingProject/#/manager/openAccount');
   }
+
+  async waitForOpened() {
+    await this.page.waitForURL('/angularJs-protractor/BankingProject/#/manager/openAccount');
+}
+
+async reload() {
+  await this.page.reload();
+}
+
+async selectCustomer(customerName) {
+    await this.customerNameDropdown.selectOption(customerName);
+}
+
+async selectCurrency(currency) {
+    await this.currencyDropdown.selectOption(currency);
+}
+
+async clickProcessButton() {
+    await this.processButton.click();
+}
+
+async assertCustomerNameDropdownContainsValue(expectedValue) {
+    const currentOptionText = await this.customerNameDropdown.inputValue();
+    expect(currentOptionText).toBe(expectedValue);
+}
+async assertCurrencyDropdownContainsValue(expectedValue) {
+    const currentOptionText = await this.currencyDropdown.inputValue();
+    expect(currentOptionText).toBe(expectedValue);
```

Comment:
The methods `assertCustomerNameDropdownContainsValue` and `assertCurrencyDropdownContainsValue` use `inputValue()` to get the current option text of dropdowns. This might not work as expected for dropdowns. Consider using `this.customerNameDropdown.evaluate(node => node.value)` or a similar approach to retrieve the selected option's value.